defaults/main.yaml

```diff
@@ -76,6 +76,15 @@ ssl:
 local:
   files: ../files
 
+kafka_file_descriptor_limit: 100000
+kafka_env_file: "{{ confluent_kafka_env_file_path }}/kafka-env.sh"
+
+kafka_init_memory_heapsize_with_units: 1G
+
+kafka_max_memory_heapsize_with_units: >
+  "{{ ((ansible_memtotal_mb/1024) | root | round | int | string ) + 'G' 
+  if ((ansible_memtotal_mb/1024) | root | round | int)> 1 else '1G' }}"
```

Comment:
TIL: Heap size init default values have increased from 256M to 1G

Reply:
Yeah. I took the default values directly from the start script for kafka and zookeeper. In this case the default values are:

`if [ "x$KAFKA_HEAP_OPTS" = "x" ]; then
    export KAFKA_HEAP_OPTS="-Xmx1G -Xms1G"
fi`

Anyway thanks for approving it